t/integration/test_canvas.py

```diff
@@ -1,9 +1,11 @@
 import re
 import tempfile
+import uuid
 from datetime import datetime, timedelta
 from time import sleep
 
 import pytest
+import pytest_subtests  # noqa: F401
```

Comment:
Why do we need this?
```suggestion
```

Reply:
This is for the side-effect which allows us to use the `subtests` fixture. I could do something like
```python
import pytests_subtests as _side_effect  # noqa: F401
```
to make that more obvious perhaps. I just ^c^ved this from the other test suite where we use subtests.